force-app/main/default/lwc/relationshipMappingModalBody/relationshipMappingModalBody.html

```diff
@@ -0,0 +1,86 @@
+                        label={labelReference.femaleValue}
+                        value={femaleValue}
+                        placeholder={labelReference.comboboxPlaceholderText}
+                        options={activeRelationshipTypesComboboxVModel.options}
+                        required="true"
+                        onchange={handleFemaleValueChange}
+                        data-qa-locator={inputAttributeReference.femaleValue}
+                    ></lightning-combobox>
+                </div>
+            </template>
+            <template if:true={activeRelationshipTypesComboboxVModel}>
+                <div class="slds-col slds-size_2-of-3 slds-p-bottom_medium">
+                    <lightning-combobox
+                        id={inputAttributeReference.maleValue}
+                        name={inputAttributeReference.maleValue}
+                        label={labelReference.maleValue}
+                        value={maleValue}
+                        placeholder={labelReference.comboboxPlaceholderText}
+                        options={activeRelationshipTypesComboboxVModel.options}
+                        required="true"
+                        onchange={handleMaleValueChange}
+                        data-qa-locator={inputAttributeReference.maleValue}
+                    ></lightning-combobox>
+                </div>
+            </template>
+            <template if:true={activeRelationshipTypesComboboxVModel}>
+                <div class="slds-col slds-size_2-of-3 slds-p-bottom_medium">
+                    <lightning-combobox
+                        id={inputAttributeReference.neutralValue}
+                        name={inputAttributeReference.neutralValue}
+                        label={labelReference.neutralValue}
+                        value={neutralValue}
+                        placeholder={labelReference.comboboxPlaceholderText}
+                        options={activeRelationshipTypesComboboxVModel.options}
+                        required="true"
+                        onchange={handleNeutralValueChange}
+                        data-qa-locator={inputAttributeReference.neutralValue}
+                    ></lightning-combobox>
+                </div>
+            </template>
+            <template if:true={activeRelationshipTypesComboboxVModel}>
+                <div class="slds-col slds-size_2-of-3 slds-p-bottom_xx-large slds-p-top_medium">
+                    <lightning-input
+                        class={inputStyling}
+                        type="toggle"
+                        label={labelReference.isActive}
+                        id={inputAttributeReference.isActive}
+                        name={inputAttributeReference.isActive}
+                        onchange={handleIsActiveChange}
+                        checked={isActive}
+                        value={isActive}
+                    ></lightning-input>
+                </div>
+            </template>
```

Comment:
Do we need to reassert the same condition across all of these? I would imagine we can wrap them in a single template if:true. This is functional, but causes more checks than are necessary on the render.

Reply:
Thanks, I changed the template to use only one if:true condition